pydis_site/static/css/home/timeline.css

```
@@ -0,0 +1,3740 @@
+    background: var(--cd-color-2)
+}
+
+@media (min-width: 64rem) {
+    .cd-timeline__container::before {
+        left: 50%;
+        -webkit-transform: translateX(-50%);
+        -ms-transform: translateX(-50%);
+        transform: translateX(-50%)
+    }
+}
+
+.cd-timeline__block {
+    display: -ms-flexbox;
+    display: flex;
+    position: relative;
+    z-index: 1;
+    margin-bottom: 2em;
+    margin-bottom: var(--space-lg)
+}
+
+.cd-timeline__block:last-child {
+    margin-bottom: 0
+}
+
+@media (min-width: 64rem) {
+    .cd-timeline__block:nth-child(even) {
+        -ms-flex-direction: row-reverse;
+        flex-direction: row-reverse
+    }
+}
+
+.cd-timeline__img {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -ms-flex-negative: 0;
+    flex-shrink: 0;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    box-shadow: 0 0 0 4px hsl(0, 0%, 100%), inset 0 2px 0 rgba(0, 0, 0, 0.08), 0 3px 0 4px rgba(0, 0, 0, 0.05);
+    box-shadow: 0 0 0 4px var(--color-white), inset 0 2px 0 rgba(0, 0, 0, 0.08), 0 3px 0 4px rgba(0, 0, 0, 0.05)
+}
+
+.cd-timeline__img i {
+    font-size: 1.5em;
+    color: white;
+}
+
+.cd-timeline__img img {
+    width: 30px;
+    height: 30px;
+}
+
+@media (min-width: 64rem) {
+    .cd-timeline__img {
+        width: 60px;
+        height: 60px;
+        -ms-flex-order: 1;
+        order: 1;
+        margin-left: calc(5% - 30px);
+        will-change: transform
+    }
+
+    .cd-timeline__block:nth-child(even) .cd-timeline__img {
+        margin-right: calc(5% - 30px)
+    }
+}
+
+.cd-timeline__img--picture {
+    background-color: #7289DA;
+}
+
+.cd-timeline__img--movie {
+    background-color: hsl(356, 53%, 49%);
+    background-color: var(--cd-color-5)
+}
+
+.cd-timeline__img--location {
+    background-color: hsl(47, 85%, 61%);
+    background-color: var(--cd-color-6)
+}
+
+.cd-timeline__content {
+    -ms-flex-positive: 1;
+    flex-grow: 1;
+    position: relative;
+    margin-left: 1.25em;
+    margin-left: var(--space-md);
+    background: hsl(0, 0%, 100%);
+    background: var(--color-white);
+    border-radius: 0.25em;
+    border-radius: var(--radius-md);
+    padding: 1.25em;
+    padding: var(--space-md);
+    box-shadow: 0 3px 0 hsl(205, 38%, 89%);
+    box-shadow: 0 3px 0 var(--cd-color-2)
+}
+
+.cd-timeline__content::before {
+    content: '';
+    position: absolute;
+    top: 16px;
+    right: 100%;
+    width: 0;
+    height: 0;
+    border: 7px solid transparent;
+    border-right-color: hsl(0, 0%, 100%);
+    border-right-color: var(--color-white)
+}
+
+.cd-timeline__content h2 {
+    color: hsl(206, 21%, 24%);
+    color: var(--cd-color-1)
+}
+
+@media (min-width: 64rem) {
+    .cd-timeline__content {
+        width: 45%;
+        -ms-flex-positive: 0;
+        flex-grow: 0;
+        will-change: transform;
+        margin: 0;
+        font-size: 0.8em;
```

```suggestion
        font-size: 0.9em;
```

0.8 feels a touch too small, feeling like I'm straining to read it when viewing both on 1080p and 1440p displays. 

It also wasn't suitable to simply use zoom, as the size disparity between the timeline card title/content makes it look comical on zooming, along with it then needing to be adjusted between other pages and this timeline, as the general body text in the rest of the site doesn't need to be zoomed.